### PR TITLE
CI: Add ci-management CI jobs

### PR DESCRIPTION
**Pull request by jwagantall**
Add basic CI jobs for ci-management repo.
Add also a centos based node dedicated for
these jobs.

Signed-off-by: Jessica Wagantall <jwagantall@linuxfoundation.org>

### CODE REVIEW

**Comment by jwagantall** [comment]
recheck

**Comment by jwagantall** [comment]
recheck

File: jjb/defaults.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    submodule-recursive: true
+    submodule-disable: false
+
+    # Jenkins
+    jenkins-ssh-credential: "jenkins"
+    jenkins-ssh-release-credential: "magma-release"
+
+    # SonarCloud
+    sonarcloud_project_organization: magma
+    sonarcloud_api_token: 846aa184392cd5f78dc3f073ace0a708198b3bb3
```
Comment (fabiopalumbo) [inline jjb/defaults.yaml:61]: Should this token be on plain text?

**Review by fabiopalumbo — APPROVED** [review]

**Comment by jwagantall** [comment]
recheck

**Comment by jwagantall** [comment]
recheck

**Comment by jwagantall** [comment]
remerge

**Pull request merged**